docs/create_usb_boot_disks.rst

```diff
@@ -0,0 +1,214 @@
+Create USB Boot Disks
```

Comment:
Nit: I would prefer to standardize on using "USB stick" or "USB drive" (we don't use "USB disk" anywhere else in the docs; Google indicates it's the least common of the three).